src/apps/gooddollar/shortcuts.ts

```diff
@@ -10,7 +10,7 @@ const client = createPublicClient({
 
 const hook: ShortcutsHook = {
   getShortcutDefinitions(_network?: string, _address?: string) {
-    return [
+    return Promise.resolve([
```

Comment:
This works, but why not turning adding the `async` keyword to the function instead?
Feels simpler.

Reply:
Meh, it felt superfluous because we don't use the await keyword.  I can change it.